src/main/java/controller/FacultyController.java

```diff
@@ -0,0 +1,50 @@
+package controller;
+
+import model.Faculty;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import service.FacultyService;
+
+import java.util.Collection;
+
+@RequestMapping("/Faculty")
+@RestController
+public class FacultyController {
+    private final FacultyService service;
+
+    public FacultyController(FacultyService service) {
+        this.service = service;
+    }
+    @PostMapping
+    public Faculty add(@RequestBody Faculty faculty){
+        return service.add(faculty);
+    }
+    @GetMapping
+    public Faculty get(@RequestParam long id){
+        return  service.get(id);
+    }
+    @PutMapping
+    public Faculty update(@RequestBody Faculty faculty){
+        return update(faculty);
+    }
+    @DeleteMapping
+    public ResponseEntity delete (@RequestParam long id){
+        service.delete(id);
+        return ResponseEntity.ok().build();
+    }
+    @GetMapping
```

Comment:
Придумай уникальный путь методу. Например:
```suggestion
    @GetMapping("/all")
```